src/litserve/utils.py

```diff
@@ -78,3 +79,11 @@ async def call_after_stream(streamer: AsyncIterator, callback, *args, **kwargs):
         logger.exception(f"Error in streamer: {e}")
     finally:
         callback(*args, **kwargs)
+
+
+@dataclasses.dataclass
+class WorkerSetupStatus:
```

Comment:
lets use strEnum https://github.com/Lightning-AI/utilities/blob/main/src/lightning_utilities/core/enums.py

Reply:
what would be the benefit of using Enum over here?

Reply:
Becase you are using the data class as enum already